.github/workflows/ci.yml

```
@@ -73,27 +73,22 @@ jobs:
 
-    - name: Install dependencies with uv
-      run: |
-        uv pip install --system -e ".[dev]"
+    - name: Run pre-commit hooks
+      uses: pre-commit/action@v3.0.1
 
-    - name: Check code formatting with Ruff
-      run: |
-        ruff format --check --diff .
+    - name: Install uv (for mypy)
+      uses: astral-sh/setup-uv@1e862dfacbd1d6d858c55d9b792c756523627244 # v7.1.4
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)